apps/wakunode2/app.nim

```diff
@@ -625,7 +625,7 @@ proc startNode(node: WakuNode, conf: WakuNodeConf,
   # retrieve px peers and add the to the peer store
   if conf.peerExchangeNode != "":
     let desiredOutDegree = node.wakuRelay.parameters.d.uint64()
-    await node.fetchPeerExchangePeers(desiredOutDegree)
+    discard await node.fetchPeerExchangePeers(desiredOutDegree)
```

Comment:
I would add a validation at this point and, in case of an error, stop the node immediately and describe the error message.
      
```suggestion
    (await node.fetchPeerExchangePeers(desiredOutDegree)).isOkOr:
      error "error while fetching peers from peer exchange", error = error 
      quit(QuitFailure)
```

Reply:
Added. Thanks for the suggestion :)